Extensions/PlatformBehavior/PlatformerObjectBehavior.cpp

```diff
@@ -19,99 +19,99 @@ This project is released under the MIT License.
 #include "PlatformBehavior.h"
 
 void PlatformerObjectBehavior::InitializeContent(
-    gd::SerializerElement& behaviorContent) {
-  behaviorContent.SetAttribute("gravity", 1000);
-  behaviorContent.SetAttribute("maxFallingSpeed", 700);
-  behaviorContent.SetAttribute("ladderClimbingSpeed", 150);
-  behaviorContent.SetAttribute("acceleration", 1500);
-  behaviorContent.SetAttribute("deceleration", 1500);
-  behaviorContent.SetAttribute("maxSpeed", 250);
-  behaviorContent.SetAttribute("jumpSpeed", 600);
-  behaviorContent.SetAttribute("jumpSustainTime", 0.2);
-  behaviorContent.SetAttribute("ignoreDefaultControls", false);
-  behaviorContent.SetAttribute("slopeMaxAngle", 60);
-  behaviorContent.SetAttribute("canGrabPlatforms", false);
-  behaviorContent.SetAttribute("canGrabWithoutMoving", true);
-  behaviorContent.SetAttribute("yGrabOffset", 0);
-  behaviorContent.SetAttribute("xGrabTolerance", 10);
-  behaviorContent.SetAttribute("useLegacyTrajectory", false);
-  behaviorContent.SetAttribute("canGoDownFromJumpthru", true);
+	gd::SerializerElement& behaviorContent) {
+	
+	behaviorContent.SetAttribute("ignoreDefaultControls", false);
+	
+	behaviorContent.SetAttribute("gravity", 1000);
+	behaviorContent.SetAttribute("jumpSpeed", 600);
+	behaviorContent.SetAttribute("jumpSustainTime", 0.2);
+	behaviorContent.SetAttribute("maxFallingSpeed", 700);
+	behaviorContent.SetAttribute("useLegacyTrajectory", false);
+	
+	behaviorContent.SetAttribute("ladderClimbingSpeed", 150);
+	
+	behaviorContent.SetAttribute("canGrabPlatforms", true);
+	behaviorContent.SetAttribute("canGrabWithoutMoving", true);
+	behaviorContent.SetAttribute("xGrabTolerance", 10);
+	behaviorContent.SetAttribute("yGrabOffset", 0);
+	
+	behaviorContent.SetAttribute("canGoDownFromJumpthru", true);
+	behaviorContent.SetAttribute("acceleration", 1500);
+	behaviorContent.SetAttribute("deceleration", 1500);
+	behaviorContent.SetAttribute("maxSpeed", 250);
+	behaviorContent.SetAttribute("slopeMaxAngle", 60);
 }
 
-#if defined(GD_IDE_ONLY)
-std::map<gd::String, gd::PropertyDescriptor>
-PlatformerObjectBehavior::GetProperties(
-    const gd::SerializerElement& behaviorContent) const {
-  std::map<gd::String, gd::PropertyDescriptor> properties;
-
-  properties[_("Gravity")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("gravity")));
-  properties[_("Jump speed")].SetGroup(_("Jump")).SetValue(
-      gd::String::From(behaviorContent.GetDoubleAttribute("jumpSpeed")));
-  properties["jumpSustainTime"]
+#
+if defined(GD_IDE_ONLY)
+std::map < gd::String, gd::PropertyDescriptor >
```

Comment:
Please run clang-format or the formatter from VS Code on this file because the code style of the file is not followed :) Autoformatting should always be used when you can.

Reply:
@4ian Thank you for the review. I apologize for all the mistakes.
I'm new to working with the code and I just dove into it. I don't think I understood your first comment, sorry. What do you mean by `SetLabel(_("Translatable name")) should be used instead.`? What should I use `SetLabel` instead of?
I will fix these issues soon! :)